src/domain/httpsactuatorserver.py

```diff
@@ -0,0 +1,95 @@
+"""HTTPS Actuator server module"""
+
+import json
+import logging
+import re
+import ssl
+import threading
+from http import HTTPStatus
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+from actuator import Actuator
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPActuatorRequestHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if re.search("/api/v1/actuators/*", self.path):
+            logger.info(
+                "Connected remote actuator from %s:%s", self.client_address[0], self.client_address[1]
+            )
+
+            record_id = self.path.split("/")[-1]
+
+            if record_id in Actuator.actuators_pool.keys():
```

Comment:
.keys() is redundant (an iterator/view over the keys is implicitly generated in those cases).